backend/user_controller.py

```diff
@@ -0,0 +1,53 @@
+from __main__ import app, supabase
```

Comment:
In the controller file, we can import app and supabase to link the main app to the route controller to make use of all the required functionality